pollination/honeybee_radiance/glare.py

```diff
@@ -0,0 +1,90 @@
+from dataclasses import dataclass
+from pollination_dsl.function import Function, command, Inputs, Outputs
+
+
+@dataclass
+class DCGlareGlareAutonomy(Function):
```

Comment:
What do you think about naming this one `DCGlareDGA` to have a more consistent naming?

Reply:
I changed the naming and dependency of the newest version of `ladybug-tools/honeybee-radiance`.